administrator/templates/atum/scss/blocks/_sidebar-nav.scss

```
@@ -19,12 +19,22 @@
       text-decoration: none;
 
       &::before {
-        margin-inline-end: .5rem;
+        margin-right: .5rem;
         font-family: "Font Awesome 5 Free";
         font-weight: 900;
         content: "\f054";
 
         [dir="rtl"] & {
+          content: none;
+          margin-right: 0;
```

```suggestion
          margin-right: 0;
          content: none;
```
Fix SSC CS error reported by Drone.